float32 scale_factor
---
# No response